packages/testcontainers/src/test-container.ts

```diff
@@ -60,9 +61,18 @@ export interface StopOptions {
   removeVolumes: boolean;
 }
 
+export interface CommitOptions {
+  repo?: string;
+  tag?: string;
```

Comment:
What's the use case for these to be undefined?

Reply:
I don't think there are any, so I've updated so they're required